scheduler/src/clients/storage-client.ts

```diff
@@ -45,9 +45,9 @@ export async function createStructure (pipelineId: number): Promise<void> {
       }
     )
   } catch (e) {
-    if (e.response.data.code === '42P07') {
+    if (e.response.status === 409 || e.response.data.code === '42P07') {
```

Comment:
Somehow a case that the previous error handling didn't catch but effectively means the same